lib/circuitry.rb

```diff
@@ -1,4 +1,4 @@
-require 'circuitry/version'
```

Comment:
I think we should still include versioning here.  Being able to type `GemName::VERSION` is a way that many users can quickly tell what version they're using.